src/components v2/Atoms/Components/VSelect/Indicator/Indicator.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <span v-bind="attributes">

```

Comment:
**issue:** no generic v-bind.

we don't know what kind of attributes we spread on here. Lets be more explicit in what we're binding here.